WcaOnRails/app/models/committee.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Committee < ActiveRecord::Base
+  has_many :teams
+  has_many :team_members, through: :teams
+  has_many :committee_positions
```

Comment:
I guess we want to delete committee positions if the committee is removed, so it makes sense to add `, dependent: :destroy` here.